tests/scripts/remote_monitoring_tests/stress_test/json/ns_input.json

```diff
@@ -0,0 +1,115 @@
+[
+ {
+    "version": "v2.0",
+    "experiment_name": "ns-demo",
+    "interval_start_time": "2022-01-23T18:25:43.511Z",
+    "interval_end_time": "2022-01-23T18:40:43.602Z",
+    "kubernetes_objects": [
+      {
+        "namespaces": 
+          {
+            "namespace": "default",
+            "metrics": [
+              {
+                "name": "namespaceCpuRequest",
+                "results": {
+                  "aggregation_info": {
+                    "sum": 6,
+                    "format": "cores"
+                  }
+                }
+              },
+              {
+                "name": "namespaceCpuLimit",
+                "results": {
+                  "aggregation_info": {
+                    "sum": 4.5,
+                    "format": "cores"
+                  }
+                }
+              },
+              {
+                "name": "namespaceCpuUsage",
+                "results": {
+                  "aggregation_info": {
+                    "min": 0.14,
+                    "max": 0.84,
+                    "avg": 0.42,
+                    "format": "cores"
+                  }
+                }
+              },
+              {
+                "name": "namespaceCpuThrottle",
+                "results": {
+                  "aggregation_info": {
+                    "min": 0.01,
+                    "max": 0.09,
+                    "avg": 0.037,
+                    "format": "cores"
+                  }
+                }
+              },
+              {
+                "name": "namespaceMemoryRequest",
+                "results": {
+                  "aggregation_info": {
+                    "sum": 400,
+                    "format": "MiB"
+                  }
+                }
+              },
+              {
+                "name": "namespaceMemoryLimit",
+                "results": {
+                  "aggregation_info": {
+                    "sum": 600,
+                    "format": "MiB"
+                  }
+                }
+              },
+              {
+                "name": "namespaceMemoryUsage",
+                "results": {
+                  "aggregation_info": {
+                    "min": 60,
+                    "max": 180,
+                    "avg": 125,
+                    "format": "MiB"
+                  }
+                }
+              },
+              {
+                "name": "namespaceMemoryRSS",
+                "results": {
+                  "aggregation_info": {
+                    "min": 55,
+                    "max": 160,
+                    "avg": 120,
+                    "format": "MiB"
+                  }
+                }
+              },
+              {
+                "name": "namespaceTotalPods",
+                "results": {
+                  "aggregation_info": {
+                    "sum": 25
```

Comment:
Currently we are supporting only avg and max aggregate metrics for namespaceTotalPods and namespaceRunningPods as part of the yaml.

Reply:
Thanks @kusumachalasani I have updated it.